Foreground.skin.php

```diff
@@ -152,36 +155,42 @@ public function execute() {
 		<?php if ($wgForegroundFeatures['NavWrapperType'] != '0') echo "</div>"; ?>
 		
 		<div id="page-content">
-		<div class="row">
-				<div class="large-12 columns">
-				<!--[if lt IE 9]>
-				<div id="siteNotice" class="sitenotice panel radius"><?php echo $this->text('sitename') . ' '. wfMessage( 'foreground-browsermsg' )->text(); ?></div>
-				<![endif]-->
-
-				<?php if ( $this->data['sitenotice'] ) { ?><div id="siteNotice" class="sitenotice"><?php $this->html( 'sitenotice' ); ?></div><?php } ?>
-				<?php if ( $this->data['newtalk'] ) { ?><div id="usermessage" class="newtalk panel radius"><?php $this->html( 'newtalk' ); ?></div><?php } ?>
-				</div>
-		</div>
+			<div class="row">
+					<div class="large-12 columns">
+					<!--[if lt IE 9]>
+					<div id="siteNotice" class="sitenotice panel radius"><?php echo $this->text('sitename') . ' '. wfMessage( 'foreground-browsermsg' )->text(); ?></div>
+					<![endif]-->
+
+					<?php if ( $this->data['sitenotice'] ) { ?><div id="siteNotice" class="sitenotice"><?php $this->html( 'sitenotice' ); ?></div><?php } ?>
+					<?php if ( $this->data['newtalk'] ) { ?><div id="usermessage" class="newtalk panel radius"><?php $this->html( 'newtalk' ); ?></div><?php } ?>
+					</div>
+			</div>
 
-		<div id="mw-js-message" style="display:none;"></div>
+			<div id="mw-js-message" style="display:none;"></div>
 
-		<div class="row">
+			<div class="row">
 				<div id="p-cactions" class="large-12 columns">
-					<?php if ($wgUser->isLoggedIn() || $wgForegroundFeatures['showActionsForAnon']): ?>
-						<a id="actions-button" href="#" data-dropdown="actions" data-options="align:left; is_hover: true; hover_timeout:700" class="button small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
+					<?php if ($wgUser->isLoggedIn() || $wgForegroundFeatures['showActionsForAnon']) { ?>
+						<a id="actions-button" href="#" data-dropdown="actions" data-options="align:bottom; is_hover: true; hover_timeout:500" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
```

Comment:
still breaking with the dropdown not reorienting to the right.

![dropdownalignment](https://cloud.githubusercontent.com/assets/3527469/12052914/e51f8eb8-aee1-11e5-885f-8d8622dbaba3.PNG)

Even if we drop the max-width to 150px it still doesn't look right

![narrow150px](https://cloud.githubusercontent.com/assets/3527469/12052926/098fb1a6-aee2-11e5-81e3-64f096ecd5f8.PNG)

I really want it to be a drop under aligned correctly, but seems as if Zurb is on to 6 and 5 is a memory now. I spent many hours trying different things and researching for the fix. The callback in js is just broken detecting the right edge.